partitura/io/importkern.py

```diff
@@ -546,6 +546,9 @@ def parse_kern(kern_path: PathLike) -> np.ndarray:
     """
     with open(kern_path, encoding="cp437") as file:
         lines = file.read().splitlines()
+    if lines[0][0] == "<":
```

Comment:
Do we need this check? With the new `load_score` there should not be any issue. Alternatively, we could cover other cases and not just only MusicXML/MEI